src/game_view.cpp

```diff
@@ -204,7 +203,7 @@ void game_view::draw_players() noexcept //!OCLINT too long indeed, please
     }
 }
 
-void game_view::set_player_coords_view() noexcept
+void game_view::set_player_info_view() noexcept
```

Comment:
Definitely a good idea to refactor this into a name that is more general and likely to match future changes in the function!